README.md

```diff
@@ -15,6 +15,13 @@
 Parley provides an API for implementing rich text layout.
 It is backed by [Swash](https://github.com/dfrg/swash).
 
+## Minimum supported Rust Version (MSRV)
+
+This version of Parley has been verified to compile with **Rust 1.70** and later.
+
+Future versions of Parley might increase the Rust version requirement.
+It will not be treated as a breaking change and as such can even happen with small patch releases.
```

Comment:
When doing testing I saw that the following error is given:

```sh
error: package `peniko v0.1.0` cannot be built because it requires rustc 1.70 or newer, while the currently active rustc version is 1.67.1
Either upgrade to rustc 1.70 or newer, or use
cargo update -p peniko@0.1.0 --precise ver
where `ver` is the latest version of `peniko` supporting rustc 1.67.1
```

Which is basically the same instructions that we had. Thus I thought this is an opportunity to simplify the readme by removing the section. I would also sync this removal with other repos eventually.